src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/UnrealPane.kt

```diff
@@ -55,12 +58,28 @@ class UnrealPane(val model: Any, lifetime: Lifetime, val project: Project) : Sim
 
         val topPanel = JPanel(HorizontalLayout(0))
         topPanel.add(topToolbar)
+
+        filter = SearchReplaceComponent.buildFor(project, consoleView.editor.contentComponent).withDataProvider(consoleView).build()
+        for (component in filter.components) {
+            if (component is OnePixelSplitter) {
+                for (innerComponent in component.components) {
+                    if (innerComponent is NonOpaquePanel || innerComponent !is JPanel) {
+                        component.remove(innerComponent)
+                    }
+                }
+                break
+            }
+        }
```

Comment:
Honestly, it would be better for us to add an ability to customize this component in the IntelliJ platform.

Reply:
This code is there to hide unused parts of the search panel.
It actually looks ugly and I can agree that it could be good to customize it in the platform.
@DecoyRS Do you think it's worth the effort?

Reply:
Right now it looks horrendous) I think everyone could benefit from making this panel a little more customizible out of the box. I agree with @ForNeVeR here.